.templates/new-module/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "[-replace-name-]",
+	"name": "@gov.au/[-replace-name-]",
```

Comment:
nope, We got'zz the dot'zz!
Proof here: https://twitter.com/npm_support/status/819316199083347968
and here: https://www.npmjs.com/org/gov.au